apps/src/dance/Dance.js

```diff
@@ -443,8 +459,6 @@ Dance.prototype.reset = function () {
 
   Sounds.getSingleton().stopAllAudio();
 
-  this.nativeAPI.reset();
```

Comment:
Does this cause any issues on levels that have preview disabled? Seems like `nativeAPI.reset` never gets called?

Reply:
It doesn't becaue there's a sneaky call to `reset` in the `playSound` callback (`onEnded`) function. It was actually causing an issue with the preview - sometimes when clicking on the reset button, the preview wouldn't be displayed because this callback was executed after the `draw` call. So I removed it in this `dance-party` [PR](https://github.com/code-dot-org/dance-party/pull/651) if the preview is on.

But now that you brought this up. I think I'll re-add `reset` call here and remove the condition in the `nativeAPI`. Thanks!